daemon/openrazer_daemon/hardware/keyboards.py

```diff
@@ -1341,6 +1341,24 @@ class RazerBlade15Advanced2021(_RippleKeyboard):
 
     DEVICE_IMAGE = "https://assets.razerzone.com/eeimages/support/products/1778/1778-razerblade15advanced2021rz09-0409x-2.png"
 
+class RazerBlade142021(_RippleKeyboard):
+    """
+    Class for the Razer Blade 14 (2021)
+    """
+    EVENT_FILE_REGEX = re.compile(r'.*Razer_Blade(-if01)?-event-kbd')
+
+    USB_VID = 0x1532
+    USB_PID = 0x0270
+    HAS_MATRIX = True
+    MATRIX_DIMS = [6, 16]
+    METHODS = ['get_device_type_keyboard', 'get_logo_active', 'set_logo_active', 'set_wave_effect', 'set_static_effect', 'set_spectrum_effect',
+               'set_reactive_effect', 'set_none_effect', 'set_breath_random_effect', 'set_breath_single_effect',
+               'set_breath_dual_effect', 'set_custom_effect', 'set_key_row',
+               'set_starlight_random_effect', 'set_starlight_single_effect', 'set_starlight_dual_effect',
+               'set_ripple_effect', 'set_ripple_effect_random_colour']
+
+
+    DEVICE_IMAGE = ""
```

Comment:
How about https://dl.razerzone.com/Images/blade15advanced%282021%29/Razer%20Blade%2015%20Advanced5_00001.png for the picture?

Reply:
@z3ntu  thats the blade 15 :smile: should be:
https://images.anandtech.com/doci/16757/Razer%20Blade%2014%20-%20%282021%29%20-%203.jpg

Reply:
@z3ntu  @BeanieBen9990  Made a pull request for this change.